### DISCUSSION TITLE
Currently, Dify's workflow applications do not seem to report data to Langsmith or Langfuse, right?

### DISCUSSION THREAD

**Discussion by YuJian920** [langgenius/dify#12189] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

In the process of using Dify, I found that when the workflow application I created runs through the complete process with Langsmith monitoring enabled, no information about the workflow call appears on Langsmith. Is this correct, or is there an issue with my configuration?`

**Comment by Yang-yyf** [comment]
When I use langfuse, It seem's everything is right. Maybe check your config is helpful

**Reply by YuJian920** [reply]
![image](https://github.com/user-attachments/assets/61dcd1eb-58ad-4707-a597-8c87c8f36d27)
My other App types are all normal, only the Workflows type won't report to langfuse😭

**Comment by dchang10** [comment]
I am encountering the same problem here. We were on version 0.10.0 and everything was working just fine. The moment we upgraded to 0.14.2, none of the logs are properly sent to langsmith. The configs were not changed at all.
